notebooks/nessie-iceberg-flink-demo-nba.ipynb

```diff
@@ -11,7 +11,7 @@
     "Initialize PyFlink\n",
     "----------------------------------------------\n",
     "To get started, we will first have to do a few setup steps that give us everything we need\n",
-    "to get started with Nessie. In case you're interested in the detailed setup steps for Flink, you can check out the [docs](https://projectnessie.org/tools/flink/)\n",
+    "to get started with Nessie. In case you're interested in the detailed setup steps for Flink, you can check out the [docs](https://projectnessie.org/tools/iceberg/flink/)\n",
```

Comment:
In the hive demo, we have useless link in the same place
we need to update it to https://projectnessie.org/tools/iceberg/hive/

Reply:
https://github.com/projectnessie/nessie-demos/blob/main/notebooks/nessie-iceberg-hive-demo-nba.ipynb

Reply:
done :)